src/types.rs

```diff
@@ -166,23 +165,29 @@ impl<S: Serialize> ToAvro for S {
 */
 
 #[derive(Debug, Clone)]
-pub struct Record {
-    pub rschema: Rc<RecordSchema>,
+pub struct Record<'a> {
```

Comment:
I have noticed that `Record` and its interface are lacking documentation. What do you think about adding it here even if it's technically outside the scope of this PR?

Reply:
Good idea! I didn't even notice :˚